web/test/helpers/helpers-formation.tsx

```diff
@@ -180,6 +180,8 @@ export type FormationPageHelpers = {
   fillAndSubmitAddressModal: (overrides: Partial<FormationSignedAddress>, fieldName: string) => Promise<void>;
   clickSubmit: () => Promise<void>;
   selectDate: (value: DateObject, fieldType: "Business start date" | "Foreign date of formation") => void;
+  uploadFile: (file: File) => void;
+  willPracticeLaw: (response?: boolean) => void;
```

Comment:
I think it'd be more readable if this function was named something that clearly described the action being performed.